src/client/pages/OfferNew/Checkout/InsuranceSummaryDetails.tsx

```diff
@@ -200,6 +203,28 @@ const getQuoteDetails = (
   return detailsGroups
 }
 
+export const getAddress = (quoteDetails: QuoteDetails) => {
+  if (!quoteDetailsHasAddress(quoteDetails)) {
+    throw new Error('Quote details need to include address field')
+  }
+
+  const { street } = quoteDetails
+
+  if ('floor' in quoteDetails && 'apartment' in quoteDetails) {
+    const { floor, apartment } = quoteDetails
+
+    if (floor || apartment) {
+      const formattedFloor = floor ? `${floor}.` : ''
+      const apartmentString = apartment ? ` ${apartment}` : ''
```

Comment:
Minor detail, just thought of the naming, should they use the same strucutre?
```suggestion
      const floorString = floor ? `${floor}.` : ''
      const apartmentString = apartment ? ` ${apartment}` : ''
```

Reply:
Hehe, thought about this but they actually aren't the same. The first one adds a `.` - meaning it's formatted differently. The other one just makes sure it's a string and not `null` or `undefined`. But maybe I'm overthinking lol 😅

Reply:
That sounds reasonable! 🙌